### ISSUE TITLE
add some more random enemy reinforcements types

### ISSUE THREAD

**Issue by JohnnyDrama69** [alganthe/Co-ops#50]
**Arma 3 Version:** `1.66` <br/>
**Co-ops version** `1.4.360` 

**Description:**
- When the AO is nearly cleared out it sometimes gets boring searching the last enemy, some random reinforcements could make it more interesting
- maybe add possibility of enemy counter attack after clearing AO
- also when side mission is finished there could be some reinforcements coming in as result

**Idea/Solution:**
- add different kind of reinforcements, randomly appearing/or as some kind of counter attack
- possible types of reinforcements: Ground inf, paradrop-infantry, truck with inf, armor reinforcements, boats

**Comment by BryanBoru** [comment]
>maybe add possibility of enemy counter attack after clearing AO

Invade & Annex had a feature like this, would be good, but consistently happening at every AO gets a bit repetitive and annoying.

**Comment by JohnnyDrama69** [comment]
yes, not at every AO, and maybe not even at the end, maybe when "clear-%" is set to 15, the reinforcement will come in at 30, so + 15%?
